pkg/kube/apisix/apis/config/v1/types.go

```diff
@@ -23,6 +23,15 @@ import (
 	"github.com/apache/apisix-ingress-controller/pkg/kube/apisix/apis/config/v2alpha1"
 )
 
+const (
+	// UseOriginalClientHost means host option set to "pass" for the upstream
+	UseOriginalClientHost = "useOriginalClientHost"
```

Comment:
As per the Kubernetes convention, the value should be in upper camel case. Like values of `ImagePullPolicy`: `Always`, `Never`, `IfNotPresent`.